config/strides/synapse-login-strides-extra.yaml

```diff
@@ -0,0 +1,14 @@
+template_path: app-extra.yaml
+stack_name: synapse-login-strides-extra
+dependencies:
+  - strides/synapse-login-strides-sageit-cert.yaml
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+parameters:
+  Listener80Arn: "arn:aws:elasticloadbalancing:us-east-1:423819316185:listener/app/awseb-AWSEB-WX1JT0B9V11G/e910ca24bdeffb8f/4faeb63ac98bd95b"
+  Listener443Arn: "arn:aws:elasticloadbalancing:us-east-1:423819316185:listener/app/awseb-AWSEB-WX1JT0B9V11G/e910ca24bdeffb8f/e38f844ce3026ee7"
+  HttpRedirectTarget: "synapse-login-strides.sagebio-strides.org"
+  FriendlyEndpoint: "strides.sc.sageit.org"
```

Comment:
this may change, it's the current leader in the naming poll: https://sagebionetworks.slack.com/archives/CEFQD0KU1/p1601929554298200

Reply:
agreed with @annagree in [@sageit](https://sagebionetworks.slack.com/archives/CEFQD0KU1/p1602261243024500) to go with `ad.strides.sc.sageit.org` and for additional host names it will be `xxx.strides.sc.sageit.org`